playbooks/README.adoc

```diff
@@ -127,3 +127,24 @@ The same approach is also possible with `include_role` but requires additionally
 ----
 
 ====
+
+== Use the verbosity parameter with debug statements
+[%collapsible]
+====
+Explanations:: Debug messages should have a verbosity defined as appropriate for the message.
+
+Rationale:: Debug messages are useful during testing and development, and can be useful to retain as playbooks go into production, but can cause concerns as they are shared between teams.
```

Comment:
I would rather say that it clutters your log with possibly a lot of needless information which might confuse users.